src/components/controller/loader.ts

```diff
@@ -1,19 +1,18 @@
+import { Data } from "../view/appView";
+
 class Loader {
-    constructor(baseLink, options) {
-        this.baseLink = baseLink;
-        this.options = options;
-    }
+    constructor(public baseLink: string, public options: object) {}
```

Comment:
```suggestion
    public baseLink: string;
    public options?: object;
  
    constructor(baseLink: string, options: object) {
        this.baseLink = baseLink;
        this.options = options;
    }
```
Ok, you use public, but anywhere need to initialize this property in Constructor.

Reply:
Agree to make it more clear :) , done